.github/workflows/load-tests-job-configs/beam_LoadTests_Go_GBK_Dataflow_Batch_100b.txt

```diff
@@ -13,8 +12,8 @@
 #  distributed under the License is distributed on an "AS IS" BASIS,
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
-# limitations under the License.
-###############################################################################
+#  limitations under the License.
+
 --job_name=load-tests-go-dataflow-batch-gbk-2-
 --project=apache-beam-testing
```

Comment:
Can we remove `--project`, `--influx_db_name` and `--influx_hostname` since they are general options and are being applied at runtime?